doc/Pacemaker_Explained/en-US/Ch-Multi-site-Clusters.txt

```diff
@@ -216,23 +215,44 @@ To revoke a ticket from this site:
 
 [IMPORTANT]
 ====
-If you are managing tickets manually. Use the `crm_ticket` command with
-great care as they cannot help verify if the same ticket is already
+If you are managing tickets manually, use the `crm_ticket` command with
+great care, because it cannot check whether the same ticket is already
 granted elsewhere. 
-
 ====
 
 
 === Granting and Revoking Tickets via a Cluster Ticket Registry ===
 
-==== Booth ====
```

Comment:
FYI @gao-yan you can see the finished product at:

old: http://clusterlabs.org/doc/en-US/Pacemaker/1.1-pcs/html/Pacemaker_Explained/ch15.html

new: http://people.redhat.com/kgaillot/doc/en-US/Pacemaker/1.1-pcs/html/Pacemaker_Explained/ch15.html